linker.ld

```diff
@@ -1,30 +1,35 @@
 OUTPUT_ARCH(arm)
 MEMORY {
-        out     : ORIGIN = 0x23D7FF0, LENGTH = 0x8010
+        /* 
+                Overlay load address + offset to common area
+                see https://docs.google.com/document/d/1Rs4icdYtiM6KYnWxMkdlw7jpWrH7qw5v6LOfDWIiYho
+        */
+        main    : ORIGIN = 0x23D7FF0, LENGTH = 0x8010
+        /* Change/Add memory locations here */
+        /* NEW    : ORIGIN = 0x2000000, LENGTH = 0x800 */
 }
 
 SECTIONS {
-        .text : {
-
-                FILL (0xABCD)
-
-                __exidx_start = .;
-                __exidx_end = .;
-                abort = .;
-
-                __text_start = . ;
-
-                __init_array_start = .;
-                KEEP (*(SORT_BY_INIT_PRIORITY(.init_array.*)))
-                KEEP (*(.init_array))
-                __init_array_end = .;
-                __fini_array_start = .;
-                KEEP (*(SORT_BY_INIT_PRIORITY(.fini_array.*)))
-                KEEP (*(.fini_array))
-                __fini_array_end = .;
-                
+        /*
+                Add more sections for the linker, each section using a different memory location
+                This allows to spread your code on different overlays
+                You can change the section of one symbol (function or variable) in your C code using: __attribute__((section(".YOURSECTIONNAME")))
+                Note that section names are standardized and in the form ".text.OV.NB.SPECIAL"
+                Where OV is between 'ov9' (ARM9 Overlay) and 'arm' (Main ARM Binary)
```

Comment:
WDYT about using `overlay` or `ov` instead of `ov9`? `ov9` looks like "overlay 9" at a glance, and there are no arm7 overlays anyway as far as I know.

Reply:
Yeah, it could be changed.
ARM7 overlays do exist but are rarely used, and not in PMD's case. If it's changed it must also be changed in `scripts/patch.py` too.